backend/WebAPI/Models/Desk.cs

```diff
@@ -0,0 +1,15 @@
+using WebAPI.Models;
+
+namespace backend.Models
+{
+    public class Desk : BaseEntity
+    {
+        public string FirstName { get; set; }
```

Comment:
Jeżeli chodzi o powiązanie między biurkiem a pracownikiem to rozbiłbym to na 2 tabele bo inaczej ciężko będzie posadzić gościa przy biurku